src/components/Form.js

```diff
@@ -1,47 +1,85 @@
 import { useForm } from 'react-hook-form';
 import styled from 'styled-components';
 import { useImmer } from 'use-immer';
-import { useState } from 'react';
+import { useState, useEffect } from 'react';
 import { DateRangeInput } from '@datepicker-react/styled';
 import { useReducer } from 'react';
 import { ThemeProvider } from 'styled-components';
+import MapboxGeocoder from '@mapbox/mapbox-gl-geocoder';
+import './Form.css';
+import mapboxgl from 'mapbox-gl';
 
 export default function Form({
   ButtonName,
```

Comment:
Only components in React should be named with a capital letter 
```suggestion
  buttonName,
```